versioned_docs/version-0.16/installation/slack/index.md

```diff
@@ -5,7 +5,7 @@ sidebar_position: 1
 ---
 
 :::info
-[Legacy Slack integration](/0.14/installation/slack/) known from previous Botkube versions has been deprecated and the old Botkube Slack app will be unapproved from the Slack App Directory on the December 21st 2022. Follow the tutorial below to use the new Slack integration.
+[Legacy Slack integration](/0.14/installation/slack/) known from previous Botkube versions has been deprecated and removed from the Slack App Directory. Follow the tutorial below to use the new Slack integration.
```

Comment:
Please also change that in 0.13, 0.14 and 0.15. Thank you!

BTW We meed to think about when to remove docs for the "archived" versions at some point 🙂

Reply:
ah.. missed that :( Will fix in next commit 👍